### ISSUE TITLE
[OpenLinkCriteria] Bug

### ISSUE THREAD

**Issue by estelleprim** [Victoire/victoire#655]
When I want to create a button or an image, I can fill all the fields expect for the "open link criteria" which is missing. It's only after saving the button and cliking another time on it in the edition mode that this criteria appears.

**Comment by alexislefebvre** [comment]
Duplicate of https://github.com/Victoire/WidgetImageBundle/issues/22.

I created https://github.com/Victoire/WidgetButtonBundle/issues/23
